src/functions.php

```diff
@@ -38,3 +40,36 @@ function usleep(int $time, LoopInterface $loop) : PromiseInterface
 {
     return Timer\resolve($time/1000000, $loop);
 }
+
+/**
+ * Returns the content type in MIME format, like text/plain or application/octet-stream, or FALSE on failure.
```

Comment:
The description is a bit wrong. It **resolves** with the content type and **rejects** on failure.

Reply:
Yes, sorry, this piece is a bit legacy